sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java

```diff
@@ -241,6 +241,10 @@ public static double getHighestSystemLoad() {
     public static void setHighestSystemLoad(double highestSystemLoad) {
         SystemRuleManager.highestSystemLoad = highestSystemLoad;
     }
+    
+    public static double getHighestCpuUsage() {
```

Comment:
Maybe it's better to be `getCpuUsageThreshold()`?

Reply:
Well